### DISCUSSION TITLE
echo插件不能用于生产环境的原因？

### DISCUSSION THREAD

**Discussion by Lewisyixin** [apache/apisix#7682] [Q&A]
https://apisix.apache.org/zh/docs/apisix/plugins/echo/
有一个想法是想把诸如 跨域策略文件(crossdomain.xml)、爬虫协议(robot.txt)这类静态文件来用echo做，但是看到官网文档写着不建议生产，能咨询下原因吗？

另外下面这个文档的中文看着应该是写错了哈~
https://apisix.apache.org/zh/docs/apisix/plugins/redirect/
![image](https://user-images.githubusercontent.com/17898688/184611874-21de4347-9422-4777-bd31-fb605e710f38.png)

**Comment by tokers** [comment]
> 另外下面这个文档的中文看着应该是写错了哈~
https://apisix.apache.org/zh/docs/apisix/plugins/redirect/

Welcome to submit a fix PR.

**Comment by tzssangglass** [comment]
> 但是看到官网文档写着不建议生产，能咨询下原因吗？

This plugin is designed to debug, not for product env.
